eth_tester/validation/inbound.py

```diff
@@ -156,7 +161,8 @@ def validate_filter_params(from_block, to_block, address, topics):
 
     invalid_topics_message = (
         "Topics must be one of `None`, an array of 32 byte hexadecimal encoded "
-        "strings, or an array of arrays of 32 byte hexadecimal strings"
+        "strings, an array of arrays of 32 byte hexadecimal strings, or an array of 32 "
+        "byte strings"
```

Comment:
I missed the "byte strings" here too. This should maybe read more like 

```py
invalid_topics_message = (
    "Topics must be one of `None`, an array of bytes with length 32 (or their hexadecimal "
    "string equivalents), or an array of arrays of bytes with length 32 (or their hexadecimal "
    "string equivalents)."
)
```

Best I could come up with 😅 but something along those lines?